README.md

```diff
@@ -6,5 +6,19 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
-1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
-1. Describe the flow of a PATCH request as it goes through the app.
+2. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+3. Describe the flow of a PATCH request as it goes through the app.
+
+
+
+1. Rails is a web framework written in Ruby.  Its used to; store information,
+for authentication, and for mulitple users to be able to interact with each other.
+
+
+2. controller decides what operation to be carried out and by whom for a user interaction.
+Model is a ruby class which is responsible to store,retrieve and validate data in databases and as well as it is used to establish relationship between different tables.
+View is responsible to render the output of user interaction back to browser in a well presented manner.
+
+3. Receive first request for the application, perfrom routing, create MVC request handler, create controller, execute controller, invoke action, execute result
```

Comment:
This is not correct.
(0. Client sends PATCH request to application)
1. Server receives PATCH request and, consulting the routes table, triggers a controller action (probably `update`) on the appropriate controller.
2. The controller action (probably `update`) triggers a method (probably also `update`) on the appropriate model.
3. The model method causes the database to update a record in the database; once successful, it returns the data that it's updated to the controller.
4. The controller takes this data and converts it to a JSON for the server to send out.
5. The server bundles up the JSON inside an HTTP response, which it sends back to the client.